kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/commands/application/slash/_Functions.kt

```diff
@@ -32,10 +32,6 @@ public suspend fun SlashCommand<*, *, *>.group(name: String, body: suspend Slash
         error("Command groups may not be nested inside subcommands.")
     }
 
-    if (subCommands.isNotEmpty()) {
```

Comment:
Isn't there also a set of subcommand functions that check whether there's a group? Please double-check that if you can, I don't have time atm

Reply:
None of the `publicSubCommand` and `ephemeralSubCommand` function in this file had a similar check (so theoretically, bots _could_ have already been using mixed groups/subcommands if they registered in the correct order).

Reply:
Oh, balls, okay. Well, thanks!